### ISSUE TITLE
How does the validation set work?

### ISSUE THREAD

**Issue by ba305** [layumi/Person_reID_baseline_pytorch#135]
Hi, would you please clarify how/why the validation set works?

I see that in prepare.py, you are using 1 image from each person in the validation set. However, in my understanding, this means that you are training on the other images from that person, so during validation, the model has already seen other images of that person, which could skew the validation accuracy.

Is there something I am misunderstanding? Thank you!

**Comment by layumi** [comment]
Hi @ba305 ,

Yes. You are right. This validation score is not so meaningful. I just use it to see whether the training is safe and sound. 

More sensible way is to split the 100 classes out of 751 classes as validation set. We could use it to conduct a similar retrieval evaluation as test setting.

**Comment by ba305** [comment]
Hi @layumi , okay, thanks for your response!
